packages/expo-font/src/FontTypes.ts

```diff
@@ -0,0 +1,16 @@
+import { Asset } from 'expo-asset';
+
+/**
+ * A font source can be a URI, a module ID, or an Expo Asset.
+ */
+export type FontSource = string | number | Asset | FontResource;
+
+// noop: web only
```

Comment:
document this more thoroughly, especially if the enum values must match web values (and link to the spec)